packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -24,9 +24,49 @@ import 'viewport.dart';
 
 export 'package:flutter/physics.dart' show Tolerance;
 
+/// Signature used by [Scrollable] to build the viewport through which the
+/// scrollable content is displayed.
 typedef Widget ViewportBuilder(BuildContext context, ViewportOffset position);
 
+/// A widget that scrolls.
+///
+/// [Scrollable] implements the interaction model for a scrollable widget,
+/// including gesture recognition, but does not have an opinion about how the
+/// viewport, which actually displays the children, is constructed.
+///
+/// It's rare to construct a [Scrollable] directly. Instead, consider [ListView]
+/// or [GridView], which combine scrolling, viewporting, and a layout model. To
+/// combine layout models (or to use a custom layout mode), consider using
+/// [CustomScrollView].
+///
```

Comment:
Maybe call attention to of() and ensureVisible() here, since those methods will not appear with ListView et al.